Zend/zend_compile.h

```diff
@@ -917,6 +917,9 @@ ZEND_API zend_string *zend_type_to_string(zend_type type);
 #define ZEND_NAME_NOT_FQ   1
 #define ZEND_NAME_RELATIVE 2
 
+/* ZEND_FETCH_ flags in class name AST of new const expression must not clash with ZEND_NAME_ flags */
+#define ZEND_CONST_EXPR_NEW_FETCH_TYPE_SHIFT 2
```

Comment:
You say it shouldn't clash but this is clashing with ``ZEND_NAME_RELATIVE`` just above.

Reply:
This is the shift amount to unclash them. :slightly_smiling_face:

Reply:
Right, again I can't read.
Although the ZEND_FETCH_CLASS constants look weird as they seem to be a bitmask in some regards but not in others due to overlapping bits. But thats irrelevant here